clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/config/AwsProviderConfig.groovy

```diff
@@ -144,33 +138,20 @@ class AwsProviderConfig {
             )
           }
         }
-
         if (credentials.discoveryEnabled) {
-          discoveryAccounts[credentials.discovery][region.name].add(credentials)
+          awsProvider.addEurekaCredential(credentials)
```

Comment:
it's probably not the right name, but this translates the AWS credential name from the accountId field in the discovery entry